pwd-service/src/main/java/org/pwd/domain/websites/Website.java

```diff
@@ -39,11 +46,15 @@ public Website(int id, URL url) {
         this.url = url;
     }
 
-    public Website(int id, Integer areaCode, URL url, String administrativeUnit, Address address) {
+    public Website(int id, Integer areaCode, URL url, String unitType,
+                   String administrativeUnit, String administrativePerson, String eMail, Address address) {
         this.id = id;
         this.areaCode = areaCode;
         this.url = url;
+        this.unitType = unitType;
         this.administrativeUnit = administrativeUnit;
+        this.administrativePerson = administrativePerson;
+        this.eMail = eMail;
```

Comment:
adminsytativePerson to będzie użyte w nagłówku maila do tej osoby. Nie
widzę potrzeby aby mieć to rozłożóne na czynniki pierwsze.

2015-09-06 0:29 GMT+02:00 Krzysztof Adamowicz <kadamowi@gmail.com>:

> zmieniłem
>
> 2015-09-05 23:48 GMT+02:00 Bartosz Gałek <notifications@github.com>:
>
>> In pwd-service/src/main/java/org/pwd/domain/websites/Website.java
>> <https://github.com/pwd-project/pwd/pull/57#discussion_r38815098>:
>>
>> >          this.administrativeUnit = administrativeUnit;
>> > +        this.administrativePerson = administrativePerson;
>> > +        this.eMail = eMail;
>>
>> email chyba bedzie lepsze niz eMail
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/pwd-project/pwd/pull/57/files#r38815098>.
>>
>
>